build-system/build-targets.js

```diff
@@ -26,9 +26,12 @@ const {gitDiffNameOnlyMaster} = require('./git');
 const ALL_TARGETS = [
   'BUNDLE_SIZE',
   'CHECKLIST',
+  'ERROR_ISSUE',
   'INVITE',
+  'ONDUTY',
   'OWNERS',
   'PR_DEPLOY',
+  'RELEASE_CALENDAR',
   'TEST_STATUS',
 ];
```

Comment:
Future cleanup idea: Make `targetMatchers` a dictionary, and replace `ALL_TARGETS` with `Object.keys(targetMatchers)`.

Inspired by https://github.com/ampproject/amphtml/pull/27297#discussion_r395138326

Reply:
I actually already had that in a branch off of this one (that's what I was referring to in the PR description), was waiting to PR it until this was merged!

Reply:
https://github.com/ampproject/amp-github-apps/pull/826